src/Resources/skeleton/security/Voter.tpl.php

```diff
@@ -8,15 +8,15 @@
 
 class <?= $class_name ?> extends Voter
 {
-    protected function supports($attribute, $subject)
+    protected function supports(<?= $use_type_hints ? 'string ' : null ?>$attribute, $subject): bool
```

Comment:
I think we have this conversation on every PR :p. Would generating the types break if I were using Symfony 4.4, where the parent class does NOT have those hints? If so, then we should do this - but only if we're on Symfony 5.

Reply:
No - i keep forgetting how this works myself, but if the abstract does not declare a return type, the child is able to declare one. For arguments it would break iirc

>Note:
>
>When overriding a parent method, the child's method must match any return type declaration on the parent. If the parent doesn't define a return type, then the child method may do so.

Reply:
>If so, then we should do this - but only if we're on Symfony 5.

The new `$use_type_hints` var checks if we're using symfony 5 or greater (double check my comparison operator :D) for the arg. But this check isnt needed for the return type.

Reply:
> No - i keep forgetting how this works myself, but if the abstract does not declare a return type, the child is able to declare one. For arguments it would break iirc...

Ah yes. It's legal to make a return type "more restrictive" than your parent, but it is NOT legal to make an argument type stricter (which would now mean you accept LESS input)